pkg/provider/resource_cluster.go

```diff
@@ -742,6 +783,7 @@ func readCluster(ctx context.Context, client *api.ClusterClient, tfClusterResour
 	tfClusterResource.FarawayReplicaIds = StringSliceToSet(responseCluster.FarawayReplicaIds)
 	tfClusterResource.PrivateNetworking = types.BoolPointerValue(responseCluster.PrivateNetworking)
 	tfClusterResource.SuperuserAccess = types.BoolPointerValue(responseCluster.SuperuserAccess)
+	tfClusterResource.PgIdentity = types.StringValue(*responseCluster.PgIdentity)
```

Comment:
this should also be defined under 
`if responseCluster.EncryptionKey != nil {}` block